<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSWindow</class>
 <widget class="QWidget" name="OSWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="backgroudWidget">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>1080</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#backgroudWidget{border-image: url(:/images/backgroud.png);}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>560</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/images/bluebutton.png);
color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="template_name_edit">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>320</y>
           <width>531</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/images/nice_light.png) 10 10 10 10;
border-top: 10px transparent;
border-right: 10px transparent;
border-bottom: 10px transparent;
border-left: 10px transparent;
color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="dns2_label_3">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>290</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;模板名称:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="dns2_label_5">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>460</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;模板描述:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="template_size_edit">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>410</y>
           <width>531</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/images/nice_light.png) 10 10 10 10;
border-top: 10px transparent;
border-right: 10px transparent;
border-bottom: 10px transparent;
border-left: 10px transparent;
color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="dns2_label_4">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>380</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;模板大小:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="dst_edit">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>490</y>
           <width>531</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/images/nice_light.png) 10 10 10 10;
background-color: rgba(255, 255, 255, 0);
border-top: 10px transparent;
border-right: 10px transparent;
border-bottom: 10px transparent;
border-left: 10px transparent;
color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'黑体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_12">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>560</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/images/bluebutton.png);
color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
        <widget class="QWidget" name="buttonLayout_2" native="true">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>100</y>
           <width>531</width>
           <height>171</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QLabel" name="buttonLabel11_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1101</width>
            <height>171</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:url(:/images/nice_light.png) 10 10 10 10;
border-top: 10px transparent;
border-right: 10px transparent;
border-bottom: 10px transparent;
border-left: 10px transparent;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>1111</width>
            <height>51</height>
           </rect>
          </property>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>10</y>
             <width>141</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;01&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>10</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;1714MB&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1111</width>
            <height>51</height>
           </rect>
          </property>
          <widget class="QCheckBox" name="checkBox_3">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>20</y>
             <width>101</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;系统序号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1101</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image:url(:/images/nice_black.png) 10 10 10 10;
border-top: 10px transparent;
border-right: 10px transparent;
border-bottom: 10px transparent;
border-left: 10px transparent;
color: #FFFFFF;
font: 12pt &quot;黑体&quot;;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>20</y>
             <width>101</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;数据大小&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <zorder>label_18</zorder>
          <zorder>label_3</zorder>
          <zorder>checkBox_3</zorder>
          <zorder>label_8</zorder>
         </widget>
        </widget>
        <widget class="QLabel" name="dns2_label_6">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>10</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;部署模式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="dns2_label_7">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;上传样机模板&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="template_name_result_label">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>290</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #ff3636;
font: 12pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;请输入模板名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lineLabel">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/line.png);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bottomWidget">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bottomWidget{border-image: url(:/images/backgroud.png);}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="hcenSpace">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="netWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="netpoint_label">
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="netstate_label">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;在线&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="ipWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>185</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="neticon_label">
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>19</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ip_label">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;192.168.255.255&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dataLabel">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;黑体&quot;;
color: #FFFFFF;</string>
        </property>
        <property name="text">
         <string>  00:00
2000/01/01</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
